src/app/utils/ProxifyUrl.js

```diff
@@ -11,6 +11,7 @@
 const rProxyDomain = /^http(s)?:\/\/steemit(dev|stage)?images.com\//g;
 const rProxyDomainsDimensions = /http(s)?:\/\/steemit(dev|stage)?images.com\/([0-9]+x[0-9]+)\//g;
 const NATURAL_SIZE = '0x0/';
+const CAPPED_SIZE = '640x0/';
```

Comment:
`import * as config from 'config';` was giving me trouble. Possibly because of a difference between server-side vs. client-side rendering? Anyone have any tips on that?